serverside_challenge_1/challenges/jibiking/app/models/electricity_charge_simulator.rb

```diff
@@ -0,0 +1,46 @@
+class ElectricityChargeSimulator
+  def calc(amp, kwh)
+    # ampが含まれるplanのidの取得
+    amperage_plan = Plan.joins(:amperages).where(amperages: {amperage: amp}).pluck(:id)
+
+    # 上記planの全kilowattoを取得
+    kilowattos = Kilowatto.joins(:plan).where(plans: {id: amperage_plan}).pluck(:min_kilowatto, :max_kilowatto, :kilowatto_price)
```

Comment:
ここもpluckを使わないで実装してほしいですね。

```suggestion
    kilowattos = Kilowatto.joins(:plan).where(plans: amperage_plan)
```